### PR TITLE
Refactor FXIOS-4399 [v103] Fix filenames for homepage and sections

### PR DESCRIPTION
**Pull request by electricRGB**
[JIRA Reference](https://mozilla-hub.atlassian.net/browse/FXIOS-4399)

### CODE REVIEW

File: Client/Application/LaunchScreen.xib [hunk 1]
```diff
@@ -1,7 +1,11 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="7706" systemVersion="14E26a" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" launchScreen="YES" useTraitCollections="YES">
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (lmarceau) [inline Client/Application/LaunchScreen.xib:1]: Why are there changes in LaunchScreen.xib?

File: Client/Frontend/Browser/TabDisplayManager.swift [hunk 2]
```diff
@@ -523,9 +523,9 @@ extension TabDisplayManager: UICollectionViewDataSource {
...
            let view = collectionView.dequeueReusableSupplementaryView(ofKind: UICollectionView.elementKindSectionHeader,
                                                                       withReuseIdentifier: GridTabViewController.independentTabsHeaderIdentifier,
-                                                                      for: indexPath) as? ASHeaderView {
+                                                                      for: indexPath) as? LabelButtonHeaderView {
 
-            let viewModel = ASHeaderViewModel(leadingInset: 15,
+            let viewModel = LabelButtonHeaderViewModel(leadingInset: 15,
                                               title: .TabTrayOtherTabsSectionHeader,
                                               titleA11yIdentifier: AccessibilityIdentifiers.TabTray.filteredTabs,
                                               isButtonHidden: true)
```
Comment (lmarceau) [inline Client/Frontend/Browser/TabDisplayManager.swift:531]: Indentation?

File: Client/Frontend/Home/JumpBackIn/JumpBackInViewModel.swift [hunk 3]
```diff
@@ -219,15 +219,15 @@ class FirefoxHomeJumpBackInViewModel: FeatureFlaggable {
...
         return .jumpBackIn
     }
 
-    var headerViewModel: ASHeaderViewModel {
-        return ASHeaderViewModel(title: FirefoxHomeSectionType.jumpBackIn.title,
+    var headerViewModel: LabelButtonHeaderViewModel {
+        return LabelButtonHeaderViewModel(title: HomepageSectionType.jumpBackIn.title,
                                  titleA11yIdentifier: AccessibilityIdentifiers.FirefoxHomepage.SectionTitles.jumpBackIn,
                                  isButtonHidden: false,
                                  buttonTitle: .RecentlySavedShowAllText,
```
Comment (lmarceau) [inline Client/Frontend/Home/JumpBackIn/JumpBackInViewModel.swift:233]: Indentation?

File: Client/Frontend/Home/Pocket/PocketViewModel.swift [hunk 4]
```diff
@@ -149,36 +149,36 @@ class FxHomePocketViewModel {
...
+    var sectionType: HomepageSectionType {
         return .pocket
     }
 
-    var headerViewModel: ASHeaderViewModel {
-        return ASHeaderViewModel(title: FirefoxHomeSectionType.pocket.title,
+    var headerViewModel: LabelButtonHeaderViewModel {
+        return LabelButtonHeaderViewModel(title: HomepageSectionType.pocket.title,
                                  titleA11yIdentifier: AccessibilityIdentifiers.FirefoxHomepage.SectionTitles.pocket,
                                  isButtonHidden: true)
```
Comment (lmarceau) [inline Client/Frontend/Home/Pocket/PocketViewModel.swift:162]: Indentation?

File: Client/Frontend/Home/RecentlySaved/RecentlySavedCellViewModel.swift [hunk 5]
```diff
@@ -95,15 +95,15 @@ class FxHomeRecentlySavedViewModel {
...
         return .recentlySaved
     }
 
-    var headerViewModel: ASHeaderViewModel {
-        return ASHeaderViewModel(title: FirefoxHomeSectionType.recentlySaved.title,
+    var headerViewModel: LabelButtonHeaderViewModel {
+        return LabelButtonHeaderViewModel(title: HomepageSectionType.recentlySaved.title,
                                  titleA11yIdentifier: AccessibilityIdentifiers.FirefoxHomepage.SectionTitles.recentlySaved,
                                  isButtonHidden: false,
                                  buttonTitle: .RecentlySavedShowAllText,
```
Comment (lmarceau) [inline Client/Frontend/Home/RecentlySaved/RecentlySavedCellViewModel.swift:109]: Indentation?

**Comment by lmarceau** [comment]
Another nit: `FX-IOS4399` instead of `FXIOS-4399` in the PR name 😄

**Comment by electricRGB** [comment]
Failing tests are the regularly failing ones. They pass locally where applicable.

**Pull request merged**